### PR TITLE
NODE-2919 Dockerfile updated to use Ubuntu Focal, so MongoDB can run on M1

### PR DESCRIPTION
**Pull request by Yavorss**

### CODE REVIEW

File: .github/workflows/build_v5_screener_imgs-test.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+name: ECR Push v5 (experimental)
```
Comment (jcolekaplan) [inline .github/workflows/build_v5_screener_imgs-test.yml:1]: I'm confused. Shouldn't this workflow be replacing our current v5 workflow: https://github.com/Contrast-Security-OSS/NodeTestBenches/blob/main/.github/workflows/build_v5_screener_imgs.yml and then rename it here and in the file name?

**Review by jcolekaplan — CHANGES_REQUESTED** [review]

File: .github/workflows/build_v5_screener_imgs-test.yml (hunk 1 above)
Comment (Yavorss) [reply .github/workflows/build_v5_screener_imgs-test.yml:1]: Yep, it's just for testing purposes here.

Comment (Yavorss) [reply .github/workflows/build_v5_screener_imgs-test.yml]: Deleted

Comment (jcolekaplan) [reply .github/workflows/build_v5_screener_imgs-test.yml]: Oh so you just need me to see if I can build these images locally?

**Pull request merged**